src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+    display: flex;
+}
+
+.stars__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+}
+
+.stars:last-child {
+    margin-right: 0;
+}
+
+.stars--4>.stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
```

Comment:
please add all variants of stars, because in future if you would like to use this layout with different cards and different ratings it will be easier to use with different numbers of stars

Reply:
all variants of stars added